tesseract-core/src/lib.rs

```diff
@@ -657,7 +657,7 @@ impl Schema {
             // Checking if we are going to sort by RCA or not
             let measure_name = s.measure.to_string();
             if let Some(ref rca) = query.rca {
-                if measure_name == format!("{} RCA", rca.mea.clone().to_string()){
+                if measure_name == format!("{} RCA", rca.mea.to_string()){
```

Comment:
since this is in the context of a format! , do we even need to the to_string()? Might be able to remove that too?

Reply:
We need to_string because the rca.mea is of format measure